pipelines/deployment.yml

```diff
@@ -1632,23 +1632,18 @@ jobs:
             repository: golang
         inputs:
           - name: rm-survey-service-source
-        outputs:
-          - name: rm-survey-service-build
-            path: rm-survey-service
         run:
           path: sh
           args:
           - -exc
           - |
-            mkdir -p /go/src/github.com/ONSdigital/
-            cp -r rm-survey-service-source /go/src/github.com/ONSdigital/rm-survey-service
-            make -C /go/src/github.com/ONSdigital/rm-survey-service
-            cp -r /go/src/github.com/ONSdigital/rm-survey-service/ .
+            cd rm-survey-service-source
```

Comment:
Are input now mutable?

Reply:
I've taken that approach at the moment to avoid having to continuously copy things back and fore though that may not strictly be best practice.

Reply:
It didn't work as expected so amending :)

Reply:
yeh inputs are not mutable :)

Reply:
:+1: